native/cocos/platform/ios/AppDelegateBridge.h

```diff
@@ -0,0 +1,36 @@
+/****************************************************************************
+ Copyright (c) 2021-2022 Xiamen Yaji Software Co., Ltd.
```

Comment:
The license's time information of new files should start from current year (2022) .

Reply:
Copyright (c) 2022 Xiamen Yaji Software Co., Ltd.

Reply:
OK